src/styles/stars.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1)::before,
+.stars--2 .stars__star:nth-child(-n + 2)::before,
+.stars--3 .stars__star:nth-child(-n + 3)::before,
+.stars--4 .stars__star:nth-child(-n + 4)::before,
+.stars--5 .stars__star:nth-child(-n + 5)::before {
```

Comment:
This part of the code is a bit repetitive. You could use a CSS preprocessor like SASS or LESS to loop through these classes and generate them dynamically. This would make your code DRY (Don't Repeat Yourself), easier to maintain and less prone to errors.